### DISCUSSION TITLE
Backup Settings to SD Card

### DISCUSSION THREAD

**Discussion by VashBaldeus** [Jyers/Marlin#420] [Ideas]
Simple, each time there's an update made, I (and I believe anyone who uses your software) has to reconfigure a lot of things.
I'd love to see an option of backing settings up onto SD card and restoring from it as well.

**Comment by Jyers** [comment]
Wait, that would be sick! Definitely will look into that.

**Reply by gothcha** [reply]
Seconded! That would reduce the need to edit the config and recompile every time for the majority of settings

**Reply by cruzer619** [reply]
3rd... *big smiley face*

**Comment by Jyers** [comment]
Check this out! https://plugins.octoprint.org/plugins/eeprom_marlin/

**Reply by VashBaldeus** [reply]
> 
> 
> Check this out! https://plugins.octoprint.org/plugins/eeprom_marlin/

Does not help people who do not have OctoPrint.

**Reply by Jyers** [reply]
Correct, but at least its something

**Reply by cruzer619** [reply]
oh nice..!!! is 1.3.0 binaries compatible with that plugin?

**Reply by Jyers** [reply]
Actually now that I check, they aren't. I'll enable EEPROM_CHITCHAT in the next release so they'll be compatible.
